### PR TITLE
Adapted for OpenRIA 5.0 rc1

### PR DESCRIPTION
**Pull request by niklas-holma**
Remove dependencies to Silverlight and net472, now only targets netstandard 2.0

Update to use on OpenRIA 5.0-rc001

Namespace changes fixes

### CODE REVIEW

File: FluentMetadata.Tests/FluentMetadata.Tests.Web/Web.config [hunk 1]
```diff
@@ -6,13 +6,13 @@
...
   <system.webServer>
     <modules runAllManagedModulesForAllRequests="true">
-      <add name="DomainServiceModule" preCondition="managedHandler" type="OpenRiaServices.DomainServices.Hosting.DomainServiceHttpModule, OpenRiaServices.DomainServices.Hosting, Version=4.0.0.0, Culture=neutral, PublicKeyToken=2e0b7ccb1ae5b4c8"/>
+      <add name="DomainServiceModule" preCondition="managedHandler" type="OpenRiaServices.Hosting.DomainServiceHttpModule, OpenRiaServices.Hosting, Version=4.0.0.0, Culture=neutral, PublicKeyToken=2e0b7ccb1ae5b4c8"/>
     </modules>
     <validation validateIntegratedModeConfiguration="false"/>
   </system.webServer>
   <system.web>
     <httpModules>
-      <add name="DomainServiceModule" type="OpenRiaServices.DomainServices.Hosting.DomainServiceHttpModule, OpenRiaServices.DomainServices.Hosting, Version=4.0.0.0, Culture=neutral, PublicKeyToken=2e0b7ccb1ae5b4c8"/>
```
Comment (Daniel-Svensson) [inline FluentMetadata.Tests/FluentMetadata.Tests.Web/Web.config:15]: I belive namespaces and names needs to be updated to include "Wcf"

File: FluentMetadata.Tests/FluentMetadata.Tests.Web/Web.config [hunk 2]
```diff
@@ -6,13 +6,13 @@
...
+      <add name="DomainServiceModule" type="OpenRiaServices.Hosting.DomainServiceHttpModule, OpenRiaServices.Hosting, Version=4.0.0.0, Culture=neutral, PublicKeyToken=2e0b7ccb1ae5b4c8"/>
     </httpModules>
     <httpRuntime targetFramework="4.6.1" />
     <compilation debug="true" targetFramework="4.6.1"/>
```
Comment (Daniel-Svensson) [inline FluentMetadata.Tests/FluentMetadata.Tests.Web/Web.config]: update targetframework to 4.7.2

File: FluentMetadata/OpenRiaServices.Server.FluentMetadata.csproj [hunk 3]
```diff
@@ -1,24 +1,18 @@
...
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
-    <PackageReference Include="System.ComponentModel.Annotations" Version="4.5.0" />
-  </ItemGroup>
-
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net45' ">
-    <Reference Include="System" />
-    <Reference Include="System.ComponentModel.DataAnnotations" />
-    <Reference Include="System.Core" />
+    <PackageReference Include="System.ComponentModel.Annotations" Version="4.7.0" />
```
Comment (Daniel-Svensson) [inline FluentMetadata/OpenRiaServices.Server.FluentMetadata.csproj]: should we update to 5.0 ?

**Review by Daniel-Svensson — CHANGES_REQUESTED** [review]
- [x] Ensure all tests passes
- [x] Ensure sample project works
- [ ] Wait for net5.0-rc1 release to nuget

File: FluentMetadata/OpenRiaServices.Server.FluentMetadata.csproj (hunk 3 above)
Comment (niklas-holma) [reply FluentMetadata/OpenRiaServices.Server.FluentMetadata.csproj]: i first thought of that but saw that OpenRiaServices.Client uses 4.7, so i assumed this was the preferred version for now

or should we update both?

**Review by Daniel-Svensson — APPROVED** [review]

**Pull request closed without merging**